Makefile

```diff
@@ -177,10 +177,16 @@ cadenceIntegTests:
 	$Q go test -v ./integ -temporal=false
 
 ci-cadence-integ-test:
-	$Q go test -v ./integ -search=false -temporal=false -dependencyWaitSeconds=180
+	$Q go test -v ./integ -search=false -temporal=false -dependencyWaitSeconds=180 -disableStickyCache=true
```

Comment:
Looks like you want to remove `disableStickyCache=true` :P

Reply:
My bad. Copying and pasting things around...